build/Headers/Headers.Shared/Internal.Shared.vcxitems

```diff
@@ -16,4 +16,9 @@
   <ItemGroup>
     <ProjectCapability Include="SourceItemsFromImports" />
   </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\UIKit\WYPopoverController.h" />
```

Comment:
my original commit to this pull actually undid someone's earlier change making _backgroundColor a StrongId. I've put that code back in the latest commit and also made better use of StrongId in UIPopoverController.mm.